src/Form/Form.js

```diff
@@ -128,6 +128,9 @@ class Form {
     get isHybrid () {
         return this.formAnalyzer.isHybrid
     }
+    get isCCForm () {
+        return this.formAnalyzer.isCCForm()
+    }
```

Comment:
The `isCCForm` was in the matching class, and was being run on every single field. `FormAnalyzer` is a more fitting place and allows us to run it once per form, rather than once per field. Massive performance improvements on pages with tons of fields.

Reply:
excellent